schemas/common/response_data.json

```diff
@@ -312,30 +312,39 @@
     "unevaluatedProperties": false,
     "required": ["feedback_type", "feedback_text", "feedback_count"]
   },
-  "feedback_response_legacy": {
+  "feedback_response": {
     "type": "object",
-    "description": "This is used for data_version 0.0.1 or 0.0.3 feedback submissions by legacy v2 runner.",
+    "description": "This is used for data_version 0.0.1 or 0.0.3 feedback submissions",
     "properties": {
-      "email": {
-        "type": "string",
-        "description": "The email of the feedback respondent"
+      "feedback_rating": {
+        "enum": ["Very easy", "Easy", "Neutral", "Difficult", "Very Difficult"],
+        "description": "The feedback rating"
       },
-      "message": {
-        "type": "string",
-        "description": "The feedback message."
-      },
-      "name": {
-        "type": "string",
-        "description": "The name of the feedback respondent."
+      "feedback_text": {
+        "$ref": "definitions.json#/non_empty_string",
+        "description": "The feedback text."
+      }
+    },
+    "additionalProperties": false,
+    "unevaluatedProperties": false,
+    "required": ["feedback_rating"]
+  },
+  "user_research_response": {
+    "type": "object",
+    "description": "This is used for data_version 0.0.1 or 0.0.3 user research submissions",
+    "properties": {
+      "full_name": {
+        "$ref": "definitions.json#/non_empty_string",
+        "description": "The full name of the feedback respondent."
```

Comment:
Only minor but these relate to user research rather than feedback so may be worth updating the descriptions